python2/koans/about_string_manipulation.py

```diff
@@ -51,26 +51,26 @@ def test_strings_can_be_split_with_different_patterns(self):
 
         words = pattern.split(string)
 
-        self.assertEqual([__, __, __, __], words)
+        self.assertEqual(["the", "rain", "in", "spain"], words)
 
         # `pattern` is a Python regular expression pattern which matches
         # ',' or ';'
 
     def test_raw_strings_do_not_interpret_escape_characters(self):
         string = r'\n'
         self.assertNotEqual('\n', string)
-        self.assertEqual(__, string)
-        self.assertEqual(__, len(string))
+        self.assertEqual("\\n", string)
+        self.assertEqual(2, len(string))
 
         # Useful in regular expressions, file paths, URLs, etc.
 
     def test_strings_can_be_joined(self):
         words = ["Now", "is", "the", "time"]
-        self.assertEqual(__, ' '.join(words))
+        self.assertEqual("Now is the time", ' '.join(words))
```

Comment:
Notice the difference between `" ".join(words)` and `"".join(words)`.

Reply:
The string at the start is the separator, right?

Reply:
That's right. If you execute 'SEP'.join(words) you'll get 'NowSEPisSEPtheSEPtime'.

Reply:
Same with any other string.